app-creator/src/utils/helpers.ts

```diff
@@ -0,0 +1,4 @@
+export function camelCaseToTitleCase(text: string) {
+  const result = text.replace(/([A-Z])/g, ' $1');
+  return result.charAt(0).toUpperCase() + result.slice(1);
+}
```

Comment:
I can't seem to find where this function is being used in this PR. Perhaps I missed it, but if it's not being used here, it's best to remove it for now.